infra/eligibility_screener/template/ecs.tf

```diff
@@ -0,0 +1,102 @@
+# security group for screener
+resource "aws_security_group" "allow-screener-traffic" {
+  name = "allow_screener_traffic"
+  description = "This rule blocks all traffic unless it is HTTPS for the eligibility screener"
+  vpc_id = "vpc-032e680f92b88bb68" # don't like that this is hardcoded; default vpc
+  # may need to update application to accept 80 then redirect.
+
+  ingress {
+    description = "TCP traffic from VPC"
+    from_port = 80
+    to_port   = 80
+    protocol = "tcp"
+    cidr_blocks = ["0.0.0.0/8"]
+  }
+
+  egress {
+    description = "allow all outbound traffic from screener"
+    from_port        = 0
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+}
+
+resource "aws_ecr_repository" "eligibility-screener-repository" {
+  name                 = "eligibility-screener-repo"
+  image_tag_mutability = "IMMUTABLE"
+}
+
+resource "aws_ecr_repository_policy" "eligibility-screener-repo-policy" {
+  repository = aws_ecr_repository.eligibility-screener-repository.name
+  policy     = <<EOF
```

Comment:
What do you think about moving this inline policy into a data source?
` tf
data "aws_iam_policy_document" "ecr-permissions" {
  statement {
    sid = "ECRPermissions"
    actions = [
      "ecr:BatchCheckLayerAvailability",
      "ecr:BatchGetImage",
      "ecr:CompleteLayerUpload",
      "ecr:GetDownloadUrlForLayer",
      "ecr:GetLifecyclePolicy",
      "ecr:InitiateLayerUpload",
      "ecr:PutImage",
      "ecr:UploadLayerPart"]
    effect = "Allow"
    principals {
      type        = "AWS"
      identifiers = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"]
    }
  }
}

resource "aws_ecr_repository_policy" "attach" {
  repository = aws_ecr_repository.ecr.name
  policy     = data.aws_iam_policy_document.ecr-permissions.json
}
`

This will allow terraform error checking to automatically format the json correctly as well as allow this policy to become reusable.